inference-engine/src/mkldnn_plugin/nodes/mkldnn_pad_node.cpp

```diff
@@ -69,12 +69,15 @@ MKLDNNPadNode::MKLDNNPadNode(const std::shared_ptr<ngraph::Node>& op, const mkld
     if (outputShapes.size() != 1)
         THROW_ERROR << "Incorrect number of output edges";
 
-    const int srcDimsRank = inputShapes[DATA_ID].getRank();
-    const int dstDimsRank = outputShapes[DATA_ID].getRank();
+    const size_t srcDimsRank = inputShapes[DATA_ID].getRank();
+    const size_t dstDimsRank = outputShapes[DATA_ID].getRank();
     if (srcDimsRank != dstDimsRank)
         THROW_ERROR << "has incorrect number of input/output dimensions!";
 
-    const auto pad = ov::as_type_ptr<const ngraph::opset1::Pad>(op);
+    auto pad = ov::as_type_ptr<const ngraph::opset1::Pad>(op);
+    if (!pad) {
+        THROW_ERROR << "supports only opset1";
```

Comment:
M.b. something more descriptive like "Couldn't cast op to opset1::Pad in Pad node with name <Name>"